src/tokenizer/stemmer.rs

```diff
@@ -1,9 +1,56 @@
-#![cfg_attr(feature = "cargo-clippy", allow(clippy::new_without_default))]
+#![cfg_attr(feature = "cargo-clippy", allow(clippy::new_without_default, doc))]
```

Comment:
Do we need that?

Reply:
Whoops, I didn't mean to leave that in, I'll remove it.